### PR TITLE
Updated Golazos metadata name and description

### PR DESCRIPTION
**Pull request by Deewai**

### CODE REVIEW

File: sport-moment-nft/contracts/Golazos.cdc [hunk 1]
```diff
@@ -585,26 +585,33 @@ pub contract Golazos: NonFungibleToken {
...
+            let matchHomeTeam: String = metadata["MatchHomeTeam"] ?? ""
+            let matchAwayTeam: String = metadata["MatchAwayTeam"] ?? ""
+            let matchHomeScore: String = metadata["MatchHomeScore"] ?? ""
+            let matchAwayScore: String = metadata["MatchAwayScore"] ?? ""
+            let matchDay: String = metadata["MatchDay"] ?? ""
+            let matchSeason: String = metadata["MatchSeason"] ?? ""
+
+            return "LaLiga Golazos Moment from ".concat(matchHomeTeam)
+            .concat(" x ").concat(matchAwayTeam).concat(" (").concat(matchHomeScore)
+            .concat("-").concat(matchAwayScore).concat(") on Matchday ")
```
Comment (sadief) [inline sport-moment-nft/contracts/Golazos.cdc:613]: Nice! What would end up displaying if the scores were 0 or null?

Comment (Deewai) [reply sport-moment-nft/contracts/Golazos.cdc:613]: should display 0-0 which is a valid scoreline. Should not have a value of null tho

Comment (sadief) [reply sport-moment-nft/contracts/Golazos.cdc:613]: 0-0 sounds good 👍  is it possible to be null at all? Like if it was a special NFT or something? (although we'd probably want to use a different contract if that was the case

Comment (Deewai) [reply sport-moment-nft/contracts/Golazos.cdc:613]: yea, also. since it's a {String:String} type, it won't accept a null value

Comment (sadief) [reply sport-moment-nft/contracts/Golazos.cdc:613]: oh nice, 👍

**Pull request merged**